docs/Reference/Collection/CollectionManipulation.md

```diff
@@ -208,6 +208,35 @@ listener.on('open',() => console.log("connection open"));
 listener.on('close',() => console.log("connection closed"));
 ```
 
+## client.enableCollecttionStream
+
+`async client.enableCollecttionStream(collection-name, hasStreamBody): Object`
+
+Updates the collection stream flag.
+
+**Arguments**
+
+- **data**: `Object`
+
+  An object with the following key-value:
+
+  - **hasStream**: `boolean` 
```

Comment:
FYI we can pass this flag while creating the collection as well. so it do we need this. Yes in the future we will need it once we can remove the stream. but now?

Reply:
I kept it so that in future we dont need to change this please let me know if we still want to change